tests/end2end-tests/graphstorm-ec/test.sh

```diff
@@ -28,7 +28,7 @@ echo "Test GraphStorm edge classification"
 date
 
 echo "**************standalone"
-python3 $GS_HOME/python/graphstorm/run/gsgnn_ep/gsgnn_ep.py --part-config /data/movielen_100k_ec_1p_4t/movie-lens-100k.json --cf $GS_HOME/training_scripts/gsgnn_ep/ml_ec.yaml
```

Comment:
DGL 2.0.0+ does not initialize `th.distributed.init_process_group(backend=backend)`, and this test does not make sense as we never claim to support the standalone mode with gsgnn_*.py.

Reply:
Yes. And it was only the previous use case before we update the standalone script. So deprecate the test here.

Reply:
Can you also update L30?